tech.json

```diff
@@ -339,9 +339,14 @@
       "techs": [
         {
           "name": "canUnmorphBombBoost",
-          "requires": ["h_canBombThings"],
+          "requires": [ "h_canBombThings" ],
           "note": "A tech that involves mid-air morphing to drop a Morph Bomb or PB, then mid-air unmorphing to briefly hover above the bomb, then using the bomb blast to go just a bit higher than max jump height"
         },
+        {
+          "name": "canWallJumpBombBoost",
+          "requires": [ "h_canBombThings" ],
+          "note": "The ability to accurately place a bomb in midair following a walljump."
```

Comment:
Can we add "canWalljump" to the requires?
And maybe make the note mention using the bomb to boost you, so it's not confused with simply placing a bomb near a bomb block at the ceiling?

Reply:
This was updated in the second commit, but the note could be clearer still

Reply:
Oh, my bad. I'm still a noob at using Github.